tests/testthat/test_euclidean_space.R

```diff
@@ -0,0 +1,19 @@
+# Unit tests for Euclidean Space
```

Comment:
I just read that this should be context("Unit tests for Euclidean Space") as per testthat conventions:
https://vita.had.co.nz/papers/testthat.pdf

Testthats "contexts" seem to be made exactly for this.

Reply:
@lundjack use "contexts"